core/src/main/java/me/leoko/advancedban/utils/Punishment.java

```diff
@@ -148,7 +148,8 @@ private void announce(int cWarnings) {
                 getType().getName() + ".Notification",
                 "OPERATOR", getOperator(),
                 "PREFIX", mi.getBoolean(mi.getConfig(), "Disable Prefix", false) ? "" : MessageManager.getMessage("General.Prefix"),
-                "DURATION", getDuration(true),
+                "DURATION", getDuration(false),
```

Comment:
You should probably keep `fromStart` here `true` as well. It might take a few ms from creating the punishment to announcing it, which can cause this to be a second less than the total time and look ugly.
Maybe also add a quick comment explaining the reasoning to prevent confusion in the future.

Reply:
This is only announced once, correct?

Good point though, will look into this right now

Reply:
Looks good:

![image](https://github.com/user-attachments/assets/b05e67b5-83ef-4231-8f60-6d80d48b638a)